### PR TITLE
WIP: unmarshalling error with nighthawk during http.runner

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: Kush Trivedi <kushthedude@gmail.com>

**Description**

This PR fix for https://github.com/layer5io/meshery/issues/1498

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: apinighthawk/nighthawk.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func NighthawkRun(config *NighthawkConfig) ([]byte, error) {
 		"--concurrency "+c,
 		"--duration "+duration,
 		rURL.String(),
-		"--output-format json").Output()
+		"--output-format fortio").Output()
```
Comment (leecalcote) [inline apinighthawk/nighthawk.go:55]: Nice. I have a chuckle each time that I see this supported by Nighthawk.

File: cmd/main.go [hunk 2]
```diff
@@ -35,6 +38,33 @@ func main() {
 		err = errors.Wrapf(err, msg)
 		log.Fatal(err)
 	}
+	res1 := string(result)
+
+	var result1 periodic.RunnerResults
+	var bd []byte
+
+	hres, _ := res1.(*fhttp.HTTPRunnerResults)
```
Comment (leecalcote) [inline cmd/main.go:46]: Hopefully, this type assertion doesn't rear its ugly head.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Should their be a separate build check?

**Comment by kushthedude** [comment]
> Should there be a separate build check?

golangci-lint does it for us, and it's failing indicates that it will fail on upstream meshery too.

File: cmd/main.go (hunk 2 above)
Comment (kushthedude) [reply cmd/main.go:46]: That is what it does in the upstream repo which ultimately caused errors in the tests and whenever there was an error in nighthawk, due to the load test interface the test fell back to fortio which is why the tests were successful each and every time.

**Pull request merged**